src/openrct2/actions/RideCreateAction.cpp

```diff
@@ -147,18 +149,8 @@ GameActions::Result RideCreateAction::Execute() const
     ride->overall_view.SetNull();
     ride->SetNameToDefault();
 
-    for (auto& station : ride->GetStations())
-    {
-        station.Start.SetNull();
-        station.Entrance.SetNull();
-        station.Exit.SetNull();
-        station.TrainAtStation = RideStation::kNoTrain;
-        station.QueueTime = 0;
-        station.SegmentLength = 0;
-        station.QueueLength = 0;
-        station.Length = 0;
-        station.Height = 0;
-    }
+    // Default initialize all stations.
+    std::ranges::fill(ride->GetStations(), RideStation{});
```

Comment:
This changes the values of `station.Start`, `station.Entrance` and `station.Exit`, which previously were set to "null", but are now either uninitialised or 0.

Reply:
You are correct, I thought it will be null by default but CoordsXY(Z) uses 0 rather than the sentinel. In the future its probably good idea to use std::optional instead of special values.

Reply:
Would it be possible to have RideStation itself default these three to the sentinel, rather than doing it in Ride CreateAction?